new eva code.py

```diff
@@ -28,6 +28,7 @@ def find(y,x,dy,dx,n,turn=False): #n is pointer (index)
     if n == 0:
         for i,m in hor+dia:
             find(y + i, x + m, i, m, n + 1, False)
+        return #QUIT here as we already stepped in all possible directions otherwise we will double step
```

Comment:
hi gang u slayed that

Reply:
thank you

Reply:
we did NOT like the comment, make it more simple 🤓☝️

Reply:
done